### ISSUE TITLE
E: Unable to locate package libarchive-tools

### ISSUE THREAD

**Issue by gustawdaniel-acaisoft** [jshimko/meteor-launchpad#144]
```
#10 38.61 Setting up ca-certificates (20141019+deb8u4) ...
#10 38.79 debconf: unable to initialize frontend: Dialog
#10 38.79 debconf: (TERM is not set, so the dialog frontend is not usable.)
#10 38.79 debconf: falling back to frontend: Readline
#10 38.79 debconf: unable to initialize frontend: Readline
#10 38.79 debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.20.2 /usr/local/share/perl/5.20.2 /usr/lib/x86_64-linux-gnu/perl5/5.20 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.20 /usr/share/perl/5.20 /usr/local/lib/site_perl .) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
#10 38.79 debconf: falling back to frontend: Teletype
#10 39.47 Updating certificates in /etc/ssl/certs... 152 added, 0 removed; done.
#10 42.34 Setting up libsasl2-modules:amd64 (2.1.26.dfsg1-13+deb8u2) ...
#10 42.35 Processing triggers for libc-bin (2.19-18+deb8u10) ...
#10 42.36 Processing triggers for ca-certificates (20141019+deb8u4) ...
#10 42.37 Updating certificates in /etc/ssl/certs... 0 added, 0 removed; done.
#10 43.20 Running hooks in /etc/ca-certificates/update.d....done.
#10 46.61 Reading package lists...
#10 50.27 Building dependency tree...
#10 50.40 Reading state information...
#10 50.45 E: Unable to locate package libarchive-tools
------
executor failed running [/bin/sh -c bash $BUILD_SCRIPTS_DIR/install-deps.sh &&          bash $BUILD_SCRIPTS_DIR/post-install-cleanup.sh]: exit code: 100
```

**Comment by gustawdaniel-acaisoft** [comment]
about: frontend: noninteractive

https://github.com/moby/moby/issues/27988

**Comment by gustawdaniel-acaisoft** [comment]
After some fixes

```
#10 99.24 Saving to: '/usr/local/bin/gosu.asc'
#10 99.24 
#10 99.24      0K                                                       100% 22.8M=0s
#10 99.24 
#10 99.24 2021-12-14 09:47:38 (22.8 MB/s) - '/usr/local/bin/gosu.asc' saved [543/543]
#10 99.24 
#10 99.25 /opt/build_scripts/install-deps.sh: line 36: gpg: command not found
```

**Comment by gustawdaniel-acaisoft** [comment]
My pr

https://github.com/jshimko/meteor-launchpad/pull/146

**Comment by jshimko** [comment]
Closed via #146
